tracking/imu/imu_lsm6ds3trc.c

```diff
@@ -1,14 +1,6 @@
-#include "lsm6ds3tr_c_reg.h"
+#include "imu_lsm6ds3trc.h"
 
-#include <furi_hal.h>
-
-#include "imu.h"
-
-#define TAG "LSM6DS3TR-C"
-
-#define LSM6DS3_ADDRESS (0x6A << 1)
-
-static const double DEG_TO_RAD = 0.017453292519943295769236907684886;
+static const double LSM6DS3_DEG_TO_RAD = 0.017453292519943295769236907684886;
```

Comment:
Nit: since the constant value and usage is the same for all three, it's probably worth moving to `imu.h` with a generic name of `DEG_TO_RAD`.